SS14.Admin/Pages/Shared/_Layout.cshtml

```diff
@@ -13,6 +13,8 @@
     <title>@ViewData["Title"] - SS14.Admin</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css"/>
     <link rel="stylesheet" href="~/css/site.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">

```

Comment:
We definitely should not rely on a CDN for stuff like this, we should bundle it with the program itself.

Reply:
I’ll fix it when I get home